lib/screens/transactions/new_trans_controller.dart

```diff
@@ -27,11 +30,23 @@ class NewTransController extends GetxController {
   RxBool isEdit = false.obs;
   Transaction? previousTransaction;
   final DbController dbController = Get.find<DbController>();
+  int? accId;
 
   @override
   void onInit() {
     super.onInit();
     isEdit.value = Get.arguments?['edit'] ?? false;
+    final cats = Utils.getCategories(
+        Get.find<DbController>().categories, categoryType.value);
+    final accounts = Get.find<DbController>()
```

Comment:
we can just filter it at once at onInit() and use the same list on multiple methods without needing to filter again.